resources/assets/js/store/attendance.js

```diff
@@ -0,0 +1,49 @@
+import Vue from 'vue'
+
+const axios = require('axios')
```

Comment:
Is axios preferable to fetch()?

Reply:
Yes.  You can do basic API calls with fetch() as you can with $.ajax, but axios lets you add things like retrying, or headers which are sent on all requests, or callbacks which intercept all your requests, or timeouts, or, or, or.